services/github-service/src/main/java/org/rhd/katapult/github/GithubResource.java

```diff
@@ -108,51 +172,52 @@ private JsonObject postToken(String code, String state, Client client)
       map.putSingle("state", state);
       // {"access_token":"3d4bf6b3ea93fba1dbfeeb5fa5afb5226e0cbec9","token_type":"bearer","scope":"public_repo,user:email"}
       JsonObject obj = client
-               .target("https://github.com/login/oauth/access_token")
-               .request()
-               .accept(MediaType.APPLICATION_JSON_TYPE)
-               .post(Entity.form(map), JsonObject.class);
+              .target("https://github.com/login/oauth/access_token")
+              .request()
+              .accept(MediaType.APPLICATION_JSON_TYPE)
+              .post(Entity.form(map), JsonObject.class);
       return obj;
    }
 
-   JsonObject forkRepository(String accessToken, String repository, Client client)
+   private JsonObject forkRepository(String accessToken, String repository, Client client)
    {
       Response response = client
```

Comment:
Can refactor this bit later to use the standalone github-service API.  For instance usage looks like:

https://github.com/redhat-developer-tooling/katapult/blob/master/services/github-service/src/test/java/org/rhd/katapult/github/impl/kohsuke/GitHubServiceIT.java#L59